File: src/main/java/com/hong/student/Student.java
package com.hong.student;

public class Student {
    String id;
    String name;
    int english;
    int math;

    public Student(String name, int english, int math) {
       this.name = name;
       this.english = english;
       this.math = math;
       }

    public void print() { //void不回傳任何型態，一呼叫就出來
        System.out.println(name + "\t" + english + "\t" + math +
                "\t" + (english + math) /2 );
    }
}

File: src/main/kotlin/com/hong/student/Student.kt
package com.hong.kotlin

fun main() {
    val stu = Student("hong", 66, 88)
    stu.print()
}

class Student(var name : String, var english : Int, var math : Int) {
 fun print() {
     println(name + "\t" + english + "\t" + math + "\t" + (english + math) /2 )
 }
}